go.mod

```diff
@@ -21,3 +20,8 @@ require (
 	github.com/spf13/viper v1.7.0
 	github.com/vulcanize/ipfs-ethdb v0.0.5
 )
+
+replace (
+	github.com/ethereum/go-ethereum v1.10.11 => github.com/Vulcanize/go-ethereum v0.0.0-20211125055606-cd7c58e7f9a2
+	github.com/vulcanize/ipfs-ethdb v0.0.5 => github.com/Vulcanize/ipfs-ethdb v0.0.0-20211125060829-0aa16344859a
```

Comment:
It doesn't seem to matter but it looks odd for Vulcanize to be capitalized in replace paths